server/routes/api/businesses.js

```diff
@@ -29,14 +27,15 @@ router.get("/getThree", (req, res) => {
 });
 
 
-router.post("/byCategory", (req, res) => {
+router.get("/getByCategory/:category", (req, res) => {
   Business.findAll({
-    where: {
-      type: req.body.category,
-    },
-  }).then((Businesses) => res.json(Businesses))
```

Comment:
Don't need to change this now but standard is to use lowercase for a parameter i.e. `then(businesses)`

Reply:
Thanks for the review